src/types.ts

```diff
@@ -628,6 +628,7 @@ export interface SettingsStorage {
   exchange: SupportedExchanges;
   debug: boolean;
   nostrEnabled: boolean;
+  tips: string[];
```

Comment:
Would it be better to restrict to actual possible strings here?

Reply:
good call out. done.

Reply:
Not sure if an `enum` is even needed maybe just an array of defined strings would be enough? Not exactly sure here. But thanks for adjusting!